src/components/field-types/DateFieldComponent.vue

```diff
@@ -81,13 +86,15 @@
         localValue: this.value,
         config: {
           wrap: true,
-          allowInput: true
+          allowInput: true,
+          enableTime: this.isTimeIncluded
```

Comment:
Why not place the `field.type === 'date-time'` here and keep the component interface simple?

Reply:
The selection of field types is something at the fieldComponent level, the DateFieldComponent does not need to know about field types, is needs to know about 'time' or 'no time'. This decouples the Datefield from the FieldComponent.